Source/WebKit/UIProcess/API/APINavigation.h

```diff
@@ -211,6 +222,9 @@ class Navigation : public ObjectImpl<Object::Type::Navigation> {
     bool m_requestIsFromClientInput { false };
     RefPtr<API::WebsitePolicies> m_websitePolicies;
     std::optional<OptionSet<WebCore::AdvancedPrivacyProtections>> m_originatorAdvancedPrivacyProtections;
+    MonotonicTime m_requestStart { MonotonicTime::now() };
+    RefPtr<WebKit::BrowsingWarning> m_safeBrowsingWarning;
+    ListHashSet<size_t> m_ongoingSafeBrowsingChecks;
```

Comment:
I feel like we may run into issues putting 0 into this ListHashSet, but there are no checks that the size_t is nonzero.

Reply:
I've confirmed inserting and removing 0 works.